devsetup/scripts/edpm-compute-bmaas.sh

```diff
@@ -0,0 +1,274 @@
+#!/bin/bash
+#
+# Copyright 2023 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+RHEL_IMAGE_URL=${RHEL_IMAGE_URL:-"https://images.rdoproject.org/centos9/master/rdo_trunk/current-tripleo/overcloud-hardened-uefi-full.qcow2"}
+AGENT_IMAGE_URL=${AGENT_IMAGE_URL:-"quay.io/openstack-k8s-operators/openstack-baremetal-operator-agent:latest"}
```

Comment:
Is it possible to use ipa image[1] here or we will use use ipa image for ironic service in overcloud's ironic.

[1] https://images.rdoproject.org/centos9/master/rdo_trunk/current-tripleo/ironic-python-agent.tar

Reply:
This is a different agent 

https://github.com/openstack-k8s-operators/openstack-baremetal-operator/tree/master/containers/agent

Reply:
thanks